README.md

```diff
@@ -52,19 +55,26 @@ Mobile Center SDK is designed with a modular approach – a developer only needs
 
 **For Xamarin.iOS and Xamarin.Android:**  
 * Navigate to the Project -> 'Add NuGet Packages...'
-* Search and select "Mobile Center Analytics" and "Mobile Center Crashes". Then Click 'Add Packages'  
+* Search for 'Mobile Center' and select "Mobile Center Analytics" and "Mobile Center Crashes". Then Click 'Add Packages'  
 
 **For Xamarin.Forms**  
 Multiplatform Xamarin.Forms app has three projects in your solution - portable class library or shared library, project.Droid, project.iOS . You need to add NuGet packages to each of these projects.
 
 * Navigate to the Project -> 'Add NuGet Packages...'
-* Search and select "Mobile Center Analytics" and "Mobile Center Crashes". Then Click 'Add Packages'
+* Search for 'Mobile Center' and select "Mobile Center Analytics" and "Mobile Center Crashes". Then Click 'Add Packages'
 
 ## For Xamarin for Visual Studio ##
 
 * Navigate Project -> Manage NuGet Packages...
-* Search and select "Mobile Center Analytics" and "Mobile Center Crashes". Then Click 'Add Packages'
+* Search for 'Mobile Center' and select "Mobile Center Analytics" and "Mobile Center Crashes". Then Click 'Add Packages'
 
+## Using Package Manager Console ##
+
+* Open Package Manager Console in either Xamarin Studio or Visual Studio  
```

Comment:
Xamarin Studio does not have package console by default unless you install an add-in (http://lastexitcode.com/blog/2014/06/22/NuGetPowerShellConsoleForXamarinStudio/), it's only Visual Studio. Should adapt wording here.